lib/Template.js

```diff
@@ -133,6 +204,15 @@ module.exports = class Template {
 		return arrayOverhead < objectOverhead ? [minId, maxId] : false;
 	}
 
+	/**
+	 *
+	 * @param {Chunk} chunk chunk whose modules will be rendered
+	 * @param {ModuleFilterPredicate} filterFn function used to filter modules from chunk to render
+	 * @param {ModuleTemplate} moduleTemplate ModuleTemplate instance used to render modules
+	 * @param {any[]} dependencyTemplates templates needed for each module to render dependencies
```

Comment:
I generally prefer to do `type TODO = any` and then only use `TODO` if it's being used as a temporary thing

Reply:
This is a really great idea, and I think this is worth it